lib/streaming.js

```diff
@@ -110,11 +119,17 @@ StreamProxy.prototype._onFinish = function() {
 StreamProxy.prototype.setStream = function(apiCall, argument) {
   var stream = apiCall(argument, this._callback);
   this.stream = stream;
-  stream.on('error', this.emit.bind(this, 'error'));
+  (['error', 'metadata', 'status']).forEach(function(event) {
+    stream.on(event, this.emit.bind(this, event));
+  }.bind(this));
+  // We also want to supply the status data as 'response'.
+  stream.on('status', this.emit.bind(this, 'response'));
```

Comment:
So if the wrapped stream emits a `status`, the StreamProxy emits both a `status` and a `response` (both set to the same value)? Why?